pkg/vms/Makefile.am

```diff
@@ -0,0 +1,90 @@
+SRCDIR=pkg/vms
```

Comment:
Using the name $(SRCDIR) is confusing as it reads like $(srcdir). Another name would be better here.